src/main/java/uk/gov/ida/matchingserviceadapter/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -13,6 +13,10 @@
     @JsonProperty
     private String hubConnectorEntityId;
 
+    @Valid
+    @JsonProperty
+    private String[] acceptableHubConnectorEntityIds;
```

Comment:
The example config files need to be changed to reflect the new way of supplying hub connector entity IDs once we know the new URL (`prod-config.yml` and `test-config.yml` and any other config files in this repo).
We'll also need to add a note to the future release notes file (`RELEASE_NOTES.md`)

Reply:
See: https://github.com/alphagov/verify-matching-service-adapter/pull/141